pkg/cloudfoundry/ManifestUtils_test.go

```diff
@@ -0,0 +1,111 @@
+package cloudfoundry
+
+import (
+	"testing"
+
+	"fmt"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadManifest(t *testing.T) {
+
+	_readFile = func(filename string) ([]byte, error) {
+		if filename == "myManifest.yaml" {
+			return []byte("applications: [{name: 'manifestAppName'}]"), nil
+		}
+		return []byte{}, fmt.Errorf("File '%s' not found", filename)
+	}
+
+	manifest, err := ReadManifest("myManifest.yaml")
+
+	appName, err := manifest.GetAppName(0)
+	if assert.NoError(t, err) {
+		assert.Equal(t, "manifestAppName", appName)
+	}
+}
+
+func TestNoRoute(t *testing.T) {
+
+	_readFile = func(filename string) ([]byte, error) {
+		if filename == "myManifest.yaml" {
+			return []byte("applications: [{name: 'manifestAppName', no-route: true}]"), nil
+		}
+		return []byte{}, fmt.Errorf("File '%s' not found", filename)
+	}
+
+	manifest, err := ReadManifest("myManifest.yaml")
+	if !assert.NoError(t, err) {
+		assert.FailNow(t, "Cannot read manifest file")
+	}
+
+	noRoute, err := manifest.GetApplicationProperty(0, "no-route")
+	if assert.NoError(t, err) {
+		noRouteAsBool, ok := noRoute.(bool)
+
+		if assert.True(t, ok) && assert.NoError(t, err) {
```

Comment:
Redundant check for err. I think you can use `assert.Equal(t, noRoute, true)`. This will also check the type.

Reply:
Code has been simplified accordingly, thanks!